Libplanet.Tests/Blockchain/BlockChainTest.cs

```diff
@@ -2173,6 +2173,37 @@ void TipChangedHandler(object target, BlockChain<DumbAction>.TipChangedEventArgs
             // TODO: Add test cases for swap
         }
 
+        [Fact]
+        private async void Reorged()
+        {
+            Block<DumbAction> actualOldTip = default;
+            Block<DumbAction> actualNewTip = default;
+            Block<DumbAction> actualBranchpoint = default;
+
+            void ReorgedHandler(object target, BlockChain<DumbAction>.ReorgedEventArgs args)
+            {
+                actualOldTip = args.OldTip;
+                actualNewTip = args.NewTip;
+                actualBranchpoint = args.Branchpoint;
```

Comment:
One thing we need to test here is how many times the event is triggered: it should be only once.  How about declaring a counter variable and increment it here?  We also need to assert the counter should be 0 until `Fork()` is called and only 1 in the end.

Reply:
Add test by https://github.com/planetarium/libplanet/pull/945/commits/cd1d870e19835906ab1b5b04b6623e9e97d6a80e